std_msgs/Header header
string obj_class
int64 ordinal_number